gatt_blacklist.txt

```diff
@@ -8,6 +8,13 @@
 # become keyloggers.
 00001812-0000-1000-8000-00805f9b34fb
 
+# Firmware update services that don't check the update's signature
+# present too much of a phishing risk to expose to the open web.
```

Comment:
LGTM

On Wed, Apr 6, 2016 at 4:31 PM, Jeffrey Yasskin <notifications@github.com>
wrote:

> In gatt_blacklist.txt
> <https://github.com/WebBluetoothCG/registries/pull/9#discussion_r58799535>
> :
>
> > @@ -8,6 +8,13 @@
> >  # become keyloggers.
> >  00001812-0000-1000-8000-00805f9b34fb
> >
> > +# Firmware update services that don't check the update's signature
> > +# present too much of a phishing risk to expose to the open web.
>
> Done, thanks.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/WebBluetoothCG/registries/pull/9/files/0c7384993a7676671eee1ffd9271e440439be2de#r58799535>
>